tests/unit_tests/test_read_data.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+import pytest
+
+from gcf_data_mapper.read import read_data_file
+
+
+@pytest.fixture(
+    params=[
+        "tests/unit_tests/test_fixtures/test.json",
+        "tests/unit_tests/test_fixtures/test.csv",
+    ]
+)
+def get_test_data(request):
+    """Fixture to yield expected data structure based on file type."""
+    file_path = request.param
+    assert os.path.exists(file_path)
+    yield read_data_file(file_path)
+
+
+def test_reads_files(get_test_data):
```

Comment:
It's probably better to parameterise the test here rather than the fixture like we spoke about yesterday, as this gets rid of the conditional logic from the test :)

Reply:
Let me know if you need a hand with this, happy to call :)